.gitignore

```diff
@@ -112,3 +112,5 @@ dropseq/13bd7eb0-23a3-4898-b043-f7e98223281g.v1/E-GEOD-81904.GSM2177575_Bipolar6
 10X/pbmc8k_S1_L008_I1_001.fastq.gz
 10X/pbmc8k_S1_L008_R1_001.fastq.gz
 10X/pbmc8k_S1_L008_R1_001.fastq.gz
+
+/tmp
```

Comment:
I'd prefer to put this in your personal .gitignore, but I'm also with leaving this in.